### PR TITLE
Created the license v exception

### PR DESCRIPTION
**Pull request by BassCoder2808**
- Intended to solve #398 
- Added the new fields in the `models.py`
- Changed the `formatXML.py` to solve the issue
- Added a new field in `form.py` to accommodate the `isException` field

### CODE REVIEW

**Comment by BassCoder2808** [comment]
@rtgdk have made all the changes and fixed the conflicts, ;let me know if anything else needs to be done in this

**Comment by rtgdk** [comment]
@BassCoder2808 Can you please close #492, let's not keep duplicates open if conflicts are resolved in new one ??

File: src/app/views.py [hunk 1]
```diff
@@ -108,6 +109,11 @@ def submitNewLicense(request):
                     licenseName = form.cleaned_data['fullname']
                     licenseIdentifier = form.cleaned_data['shortIdentifier']
                     licenseOsi = form.cleaned_data['osiApproved']
+                    isException = form.cleaned_data['isException']
+                    if isException == "False":
```
Comment (rtgdk) [inline src/app/views.py:113]: What if the value given is `false`? Use it like 
`isException.lower() == "true" then true else false` instead

**Comment by rtgdk** [comment]
@BassCoder2808 Please resolve the conflict

**Comment by BassCoder2808** [comment]
Resolved

File: src/app/views.py [hunk 2]
```diff
@@ -1201,20 +1219,22 @@ def handle_pull_request(request, is_ns):
...
+                        request.POST["branchName"],
+                        request.POST["updateUpstream"],
+                        request.POST["fileName"],
+                        request.POST["commitMessage"],
+                        request.POST["prTitle"],
+                        request.POST["prBody"],
+                        request.POST["xmlText"],
+                        license_obj.text,
+                        license_obj.isException,
+                        is_ns,
```
[the remark this answers is not in the document]
Comment (rtgdk) [reply src/app/views.py:1236]: Let's pick all these nits later. @BassCoder2808 , merging it

**Review by rtgdk — APPROVED** [review]

**Pull request merged**